### PR TITLE
Updates segmetrics to most recent version and improves the UI of the tool

### PR DESCRIPTION
**Pull request by kostrykin**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/segmetrics/run-segmetrics.py [hunk 1]
```diff
@@ -1,133 +1,59 @@
...
+    with tempfile.NamedTemporaryFile() as csv_output_file:
+        cmd = ['python', '-m', 'segmetrics.cli', str(seg_dir), str(seg_file), str(gt_file), str(csv_output_file.name), '--semicolon']
+        if recursive:
+            cmd.append('--recursive')
+        if gt_unique:
+            cmd.append('--gt-unique')
+        if seg_unique:
+            cmd.append('--seg-unique')
+        cmd += measures
+        print(cmd)
```
Comment (bgruening) [inline tools/segmetrics/run-segmetrics.py:28]: this can be removed?

File: tools/segmetrics/run-segmetrics.py [hunk 2]
```diff
@@ -1,133 +1,59 @@
...
+        subprocess.run(cmd, check=True)
+        df = pd.read_csv(csv_output_file.name, sep=';')
+        df.to_csv(str(tsv_output_file), sep='\t', index=False)
+        import shutil
```
Comment (bgruening) [inline tools/segmetrics/run-segmetrics.py:32]: I guess this should move to the top

File: tools/segmetrics/segmetrics.xml [hunk 3]
```diff
@@ -13,24 +13,28 @@
...
+       #end if
+       #if $is_distance:
+           #if $m.measure_type.object_based:
+               #set $suffix = ".object_based()"
+           #end if
+       #end if
+       #if str($m.measure_type.aggregation) != "":
+           #set $kwargs = "aggregation='" + str($m.measure_type.aggregation) + "', " + $kwargs
+       #end if
+       "sm.${m.measure_type.measure_type_selector}(${kwargs})${suffix}"
```
Comment (bgruening) [inline tools/segmetrics/segmetrics.xml:36]: can we use single quotes whenever possible, this is a bit more secure

File: tools/segmetrics/run-segmetrics.py (hunk 1 above)
Comment (kostrykin) [reply tools/segmetrics/run-segmetrics.py:28]: Thanks! I forgot to remove some code temporarily used for debugging… will fix that.

File: tools/segmetrics/run-segmetrics.py (hunk 2 above)
Comment (kostrykin) [reply tools/segmetrics/run-segmetrics.py:32]: In fact, this was also only used for debugging, so I will remove it.

File: tools/segmetrics/segmetrics.xml (hunk 3 above)
Comment (kostrykin) [reply tools/segmetrics/segmetrics.xml]: I have changed this in 1e5865766fe9b9ec951cf76da33c13827c9ddaa7.

Anything else to be done before merging?

**Review by bgruening — APPROVED** [review]
Nice

**Pull request merged**